packages/api/src/types.ts

```diff
@@ -147,7 +147,7 @@ export interface QueryableStorage<ApiType> {
   [index: string]: QueryableModuleStorage<ApiType>;
 }
 
-export interface SubmittableExtrinsicFunction<ApiType> extends CallFunction {
+export interface SubmittableExtrinsicFunction<ApiType> extends Pick<CallFunction, 'callIndex' | 'meta' | 'method' | 'section' | 'toJSON'> {
```

Comment:
instead of `Pick` maybe just declare it explicitly is more clear.
```
export interface SubmittableExtrinsicFunction<ApiType> {
  (...params: CodecArg[]): SubmittableExtrinsic<ApiType>;
  callIndex: Uint8Array;
  meta: FunctionMetadataV7;
  method: string;
  section: string;
  toJSON: () => any;
}
```
only this means another place to change when doing Metadata bump